jsonnet/openebs-mixin/rules/prometheusRules.libsonnet

```diff
@@ -0,0 +1,11 @@
+local cstor = (import 'cstor-rules.libsonnet');
```

Comment:
Bit inconsistent on naming the file, cstor-rules and jiva-rules, but prometheusRules.

Reply:
Changed it to `prometheus-rules`